lib/gui/etcher.js

```diff
@@ -50,6 +49,8 @@ electron.app.on('ready', () => {
     icon: path.join(__dirname, '..', '..', 'assets', 'icon.png')
   })
 
+  buildWindowMenu(mainWindow)
```

Comment:
I'm not sure why, but I think it feels 'cleaner' to me for `./menu` to (somehow) return the menu structure / object, and then still leave it up to `etcher.js` to be responsible for actually calling `electron.Menu.setApplicationMenu` ? \*shrug\*
Although I also prefer the way that `./menu` now just returns an exported function, rather then running code automatically when you `require` it ;-)  (but again, this is just IMHO!)

Reply:
Yeah, had to do this to be able to ref the mainWindow, because the previous .getFocusedWindow() could return the devtools window when open, and thus prevent toggling it on/off